readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
I don't mind either way we go on this but let's either make all the markdown 80 characters wide or soft word-wrapped after this PR. Just let me know which way you'd like to go and I can put a PR in.